data/json/professions.json

```diff
@@ -3904,6 +3971,35 @@
       "female": { "entries": [ { "item": "sports_bra" }, { "item": "panties" } ] }
     }
   },
+  {
```

Comment:
This isn't really justifyable IMO. One, in practice computers: 7 is a lot of hacking ability that's not implied by "programner", two, the rest of the profession is just a bunch of stereotypes.